server/cli/launch.py

```diff
@@ -354,7 +353,7 @@ def launch(
     # > cellxgene launch --dataroot <url>
 
     if dump_default_config:
-        print(default_config)
+        print(yaml.dump(DEFAULT_CONFIG.default_config))
```

Comment:
I thought it would be better to just print the config vars without any cruft. So it is a clean example file?

Reply:
"cruft" may be in the eye of the beholder :)
The comments are the documentation for the config file.
It's meant to tell the user what each config parameter is used for.
So far that's our only documentation for the config file -- unless the user looks through our git repo or the cellxgene distribution.
Maybe we can change that if we had some online documentation to describe the parameters.
But even then it might be useful to have a comment in the yaml that says "see http://... for documentation on this file"

The other issue I raised was ordering of the parameters.   I chose the ordering intentionally to have the more important ones first.   The behavior here is to alphabetize them.  Also, there are no blank lines separating sections.

Reply:
ok that makes sense. I was thinking we just wanted to give the user a clean example they could edit but if its the only way they can access the config documentation its probably better to print it all out

Reply:
But also we should prioritize moving the config documentation